src/qaqc/01-organize_raw_data.Rmd

```diff
@@ -307,57 +324,64 @@ temperature_archery <- temperature_archery %>%
 
 ```
 
-For wonky data next to each other (defined by roll sd) - here we make some seasonal
-summaries to help with this
+Site visits: 
 
 ```{r}
-temperature_summary <- temperature_archery %>% 
-  group_by(y_q) %>% 
-  summarise(yq_sd = sd(p1, na.rm = T))
+# flag the start time of a site visit
+temperature_archery <- temperature_archery %>%
+  mutate(flag = if_else((!is.na(visit_comments)), 
```

Comment:
todo: this flagging step should happen across all data (aka, earlier in the workflow) so that you don't have to run this step for every site and every parameter.